src/client/datascience/interactive-window/interactiveWindowProvider.ts

```diff
@@ -70,13 +70,17 @@ export class InteractiveWindowProvider implements IInteractiveWindowProvider, IA
     public get onDidChangeActiveInteractiveWindow(): Event<IInteractiveWindow | undefined> {
         return this._onDidChangeActiveInteractiveWindow.event;
     }
+    public get onDidCreateInteractiveWindow(): Event<IInteractiveWindow | undefined> {
```

Comment:
```suggestion
    public get onDidCreateInteractiveWindow(): Event<IInteractiveWindow> {
```

It cannot be undefined as there will always be something when creating a new window.